website/src/components/misc/page-top.tsx

```diff
@@ -9,32 +9,22 @@ export const PageTop: FunctionComponent<{ onTopScroll: () => void }> = ({
   onTopScroll,
 }) => {
   const ref = createRef<HTMLButtonElement>();
-  const showButton = useSelector<State, boolean>(
-    (state) => state.common.yScrollPosition > 60
+  const hideButton = useSelector<State, boolean>(
+    (state) => state.common.yScrollPosition <= 60
   );
 
   return (
-    <JumpToTop
-      className={showButton ? "show" : ""}
-      ref={ref}
-      onClick={onTopScroll}
-    >
+    <JumpToTop hideButton={hideButton} ref={ref} onClick={onTopScroll}>
```

Comment:
This is less performant than using a class.

Reply:
The class was causing the chrome scroll bar to stop. I think it is some kind of issue with chrome, I found older issues with similar problems in the chrome bugtracker. It works fine in all other browsers, but not in chromium. Without using the class it works fine. But also not with all css properties :D Some still reset the scrollbar state. That is also the reason for:
```
height: 0;
overflow: hidden;
box-shadow: none;
```

because setting `display: none` would cause the scrollbar to bug out again.